docs/application/native/guides/app-management/tidl.md

```diff
@@ -0,0 +1,328 @@
+
+# TIDL
+
+TIDL is programming language to define interfaces for communicating among apps in Tizen.
```

Comment:
Please use "a" before programming language.

Reply:
Thanks, I'll change it